docs/faq.md

```diff
@@ -40,4 +40,9 @@ Using the cloud provider's console, verify the bucket exists. If so, ensure that
 **Requester pays buckets**: If it is a public bucket, it may be a [requester pays bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/RequesterPaysBuckets.html). Any egress fees from this bucket will be paid by the requester (i.e. Skyplane) instead of the bucket's owner. By default Skyplane disables support for requester pays to avoid unexpected egress charges.
 
 To enable support for requester pays buckets, run `skyplane config set requester_pays true`.
-``` 
\ No newline at end of file
+``` 
+
+# How can I switch between GCP projects? 
```

Comment:
+1 this is a big place of friction, make sure the project ID is anonymized though